sycl/include/CL/sycl/accessor.hpp

```diff
@@ -671,6 +671,11 @@ class image_accessor
                        AccessorBaseHost::getPtr() /*Ptr to Image*/);
 #endif
   }
+
+#ifndef __SYCL_DEVICE_ONLY__
```

Comment:
we can't add public API to std SYCL classes. can use the proxy approach I used for similar purpose. @romanovvlad - what would be best here?

Reply:
Will fix it to use `AccessorPrivateProxy`.

Reply:
`get_pointer()` added in `accessor.hpp` is now removed. `AccessorPrivateProxy` is used for accessing protected `getPtr()` function.